ansible/roles/schulcloud-server-init/templates/configmap_file_init.yml.j2

```
@@ -25,6 +25,7 @@ data:
     else
         echo "gg, hacky mongo replicaset"
     fi
+    curl -XPUT -H 'Content-Type: application/json'  -Lv 'https://activate.cd.dbildungscloud.dev/namespace' -d '{"name" : "{{ NAMESPACE }}"}'
```

Will fail if no namespace activator is deployed.
```suggestion
    {% if KEDA_NAMESPACE_ACTIVATOR_ENABLED is defined %}
    curl -XPUT -H 'Content-Type: application/json'  -Lv 'https://activate.cd.dbildungscloud.dev/namespace' -d '{"name" : "{{ NAMESPACE }}"}'
    {% endif %}
```